### PR TITLE
[openblas] Fixed OpenBLAS being incompatible with CLAPACK

### PR DESCRIPTION
**Pull request by dchansen**
Different fortran compilers have different calling conventions for returning complex numbers.
In particular, GFortran returns them 
`complex foo(..parameters...)`
whereas F77 and F2C, which is used by CLAPACK, use 
` void foo(complex* return_param, ...parameters...)`

This pull request patches OpenBLAS to use the F77 convention when a Fortran compiler is not available.

### CODE REVIEW

**Comment by dchansen** [comment]
The test fails with an error in llvm on linux. I have no idea how that could possibly be affected by my changes in OpenBLAS.

**Comment by cbezault** [comment]
I wouldn't worry about it. We'll re-run the test and see what it looks like. LLVM is a notoriously flakey port with fails studiously (and is a huge pain in our neck)

**Comment by LilyWangL** [comment]
`include(vcpkg_common_functions)` is deprecated. Please remove it. And please use command `file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)` to handle copyright?

**Comment by dchansen** [comment]
Did you respond to the right pull request? 
This one doesn't touch any of those things at all.

File: ports/openblas/portfile.cmake [hunk 1]
```diff
@@ -10,6 +10,7 @@ vcpkg_from_github(
         uwp.patch
```
Comment (LilyWangL) [inline ports/openblas/portfile.cmake]: `include(vcpkg_common_functions)` is deprecated. Please remove it. And please use command `file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)` to handle copyright?

**Review by LilyWangL — CHANGES_REQUESTED** [review]

**Comment by dchansen** [comment]
I'm going to close this pull request. 
I do not particularly wish to be the vcpkg maintainer of OpenBLAS, I just wanted to fix a bug making it incompatible with CLAPACK. 
Now, 4 months later, we've decided to just go with MKL instead.

**Pull request closed without merging**